beta/index.markdown

```diff
@@ -0,0 +1,183 @@
+---
+layout: page
+title:  "Beta"
+show_header_link: true
+categories:
+  - bonnyci
+tags:
+  - zuul
+  - ci
+  - testing
+  - gating
+---
+
+Avast! Welcome, and thanks for your interest in BonnyCI. It's about time
+we publicly explain what BonnyCI is, and why it works the way it does.
+
+
+The story begins with [OpenStack](http://www.openstack.org/). Early
```

Comment:
I love the reference links elsewhere in the file. Should we make OpenStack one too?